warehouse/rss/views.py

```diff
@@ -72,3 +73,33 @@ def rss_packages(request):
     )
 
     return {"newest_projects": newest_projects}
+
+
+@view_config(
+    route_name="rss.user_updates",
+    context=User,
+    renderer="rss/user_updates.xml",
+    decorator=[
+        origin_cache(
+            1 * 24 * 60 * 60,  # 1 day
+            stale_while_revalidate=1 * 24 * 60 * 60,  # 1 day
+            stale_if_error=5 * 24 * 60 * 60,  # 5 days
+        )
+    ],
+)
+def rss_user_updates(user, request):
+    request.response.content_type = "text/xml"
+
+    request.find_service(name="csp").merge(XML_CSP)
+
+    latest_releases = (
+        request.db.query(Release)
+        .join(Project)
+        .filter(Project.users.any(username=user.username))
```

Comment:
Note that this doesn't mean that this user has made this release, but rather that this user has some role on a project that has made a release.

In #4603 the request is for both:

> (I actually want 2 feeds: one for "stuff I did directly" and one for "projects I am a maintainer on that did releases", to account for projects where the actual releases are done by a bot-user, or by a release manager.)

I think it's fine if we're only implementing half of #4603 here, but we might want to think a little more about what we're calling this feed since `updates` is pretty generic and we might have multiple feeds here eventually.

We also might want to make it more clear on the profile page what this feed is exactly.

Reply:
Hrm...maybe call them "My Releases" vs. "My Projects' Releases"?